Userland/Libraries/LibVideo/VP9/Decoder.cpp

```diff
@@ -693,15 +693,15 @@ MotionVector Decoder::select_motion_vector(u8 plane, BlockContext const& block_c
     auto round_mv_comp_q2 = [&](MotionVector in) {
         // return (value < 0 ? value - 1 : value + 1) / 2
         return MotionVector {
-            (in.row() < 0 ? in.row() - 1 : in.row() + 1) >> 1,
-            (in.column() < 0 ? in.column() - 1 : in.column() + 1) >> 1
+            (in.row() < 0 ? in.row() : in.row() + 1) >> 1,
```

Comment:
How about just saying `/ 2` like the spec does and keep the `- 1` and let the compiler worry about bit shifts? Or is this a measured speedup somehow?

As-is, it's weird to have a spec comment, and then code that looks very different below it.